spring-cloud-stream-binder-kinesis/src/main/java/org/springframework/cloud/stream/binder/kinesis/KinesisMessageChannelBinder.java

```diff
@@ -169,4 +169,12 @@ protected MessageProducer createConsumerEndpoint(ConsumerDestination destination
 		return adapter;
 	}
 
+	public MetadataStore getMetaDataStore() {
```

Comment:
We don't need getter and seems for me the name of the setter must be `setMetadataStore`. Since the target class is `MetadataStore`.